templates/Program/program_editor.html.twig

```diff
@@ -57,46 +57,56 @@
-                    <span class="mdc-floating-label">{{ 'footer.language'|trans({}, "catroweb") }}</span>
-                  </span>
-                  <span class="mdc-notched-outline__trailing"></span>
+        <div class="row justify-content-between" style="margin-top: 1rem;">
+          <div id="edit-language-selector" class="mdc-select mdc-select--outlined">
+            <div class="mdc-select__anchor" aria-labelledby="outlined-select-label">
+              <span class="mdc-notched-outline">
+                <span class="mdc-notched-outline__leading"></span>
+                <span class="mdc-notched-outline__notch">
+                  <span class="mdc-floating-label">{{ 'footer.language'|trans({}, "catroweb") }}</span>
                 </span>
-                <span class="mdc-select__selected-text-container">
-                  <span id="edit-selected-language" class="mdc-select__selected-text" data-selected-value="default"></span>
-                </span>
-                <span class="mdc-select__dropdown-icon">
-                  {% include 'components/mdc_select_svg.html.twig' %}
-                </span>
-              </div>
-              <div class="mdc-select__menu mdc-menu mdc-menu-surface mdc-menu-surface--fullwidth">
-                <ul id="edit-language-selector-list" class="mdc-list" role="listbox">
-                  <li class="mdc-list-item" data-value="default" role="option">
-                      <span class="mdc-list-item__ripple"></span>
-                      <span class="mdc-list-item__text">
-                        {{ "programs.default"|trans({}, "catroweb") }}
-                      </span>
-                  </li>
-                </ul>
-              </div>
+                <span class="mdc-notched-outline__trailing"></span>
+              </span>
+              <span class="mdc-select__selected-text-container">
+                <span id="edit-selected-language" class="mdc-select__selected-text" data-selected-value="default"></span>
+              </span>
+              <span class="mdc-select__dropdown-icon">
+                {% include 'components/mdc_select_svg.html.twig' %}
+              </span>
+            </div>
+            <div class="mdc-select__menu mdc-menu mdc-menu-surface mdc-menu-surface--fullwidth">
+              <ul id="edit-language-selector-list" class="mdc-list" role="listbox">
+                <li class="mdc-list-item" data-value="default" role="option">
+                    <span class="mdc-list-item__ripple"></span>
+                    <span class="mdc-list-item__text">
+                      {{ "programs.default"|trans({}, "catroweb") }}
+                    </span>
+                </li>
+              </ul>
             </div>
           </div>
+        </div>
+
+        <div class="justify-content-between" style="margin-top: 1rem; float: right;">
+          <button id="edit-delete-button" class="btn btn-primary" style="padding: 10px; background-color: red; border-color: red;">
```

Comment:
i think there is a bootstrap class that might work btn-danger

Reply:
Updated to btn-danger